tokens/typography.yml

```
@@ -2,11 +2,11 @@ props:
   # Font stacks
   - name: font-stack-base
     value: -apple-system, 'BlinkMacSystemFont', 'San Francisco', 'Roboto', 'Segoe UI', 'Helvetica Neue', sans-serif
```

Not sure why this was quoted :/

```suggestion
    value: -apple-system, BlinkMacSystemFont, 'San Francisco', 'Roboto', 'Segoe UI', 'Helvetica Neue', sans-serif
```

---

This isn’t a generic family which requires no quotes, but nice to signal it is a system font.